tests/unit/knockout/combo/tests.html

```diff
@@ -211,214 +240,222 @@
+				equal($combo.igCombo("valueInput").attr("disabled"), undefined, "Editor should be enabled");
+				ok(!$combo.igCombo("comboWrapper").hasClass($.ui.igCombo.prototype.css.disabled), "Editor should be enabled");
+				$chk.click();
+				ok($combo.igCombo("option", "disabled"), "Editor should be disabled");
+				ok($combo.igCombo("valueInput").prop("disabled"), "Editor should be disabled");
+				equal($combo.igCombo("valueInput").attr("disabled"), "disabled", "Editor should be disabled");
+				ok($combo.igCombo("comboWrapper").hasClass($.ui.igCombo.prototype.css.disabled), "Editor should be disabled");
+				$chk.click();
+				$chk.click();
+				$chk.click();
+				ok(!$combo.igCombo("option", "disabled"), "Editor should be enabled");
+				ok(!$combo.igCombo("valueInput").prop("disabled"), "Editor should be enabled");
+				equal($combo.igCombo("valueInput").attr("disabled"), undefined, "Editor should be enabled");
+				ok(!$combo.igCombo("comboWrapper").hasClass($.ui.igCombo.prototype.css.disabled), "Editor should be enabled");
+				$combo.remove();
+				$chk.remove();
+			});
 
-		var testId = 'Test disabled binding';
-		test(testId, 12, function () {
-			var $combo = $("#comboDisabled"), $chk = $("#chkDisabled");
-
-			ok(!$combo.igCombo("option", "disabled"), "Editor should be enabled");
-			ok(!$combo.igCombo("valueInput").prop("disabled"), "Editor should be enabled");
-			equal($combo.igCombo("valueInput").attr("disabled"), undefined, "Editor should be enabled");
-			ok(!$combo.igCombo("comboWrapper").hasClass($.ui.igCombo.prototype.css.disabled), "Editor should be enabled");
-			$chk.click();
-			ok($combo.igCombo("option", "disabled"), "Editor should be disabled");
-			ok($combo.igCombo("valueInput").prop("disabled"), "Editor should be disabled");
-			equal($combo.igCombo("valueInput").attr("disabled"), "disabled", "Editor should be disabled");
-			ok($combo.igCombo("comboWrapper").hasClass($.ui.igCombo.prototype.css.disabled), "Editor should be disabled");
-			$chk.click();
-			$chk.click();
-			$chk.click();
-			ok(!$combo.igCombo("option", "disabled"), "Editor should be enabled");
-			ok(!$combo.igCombo("valueInput").prop("disabled"), "Editor should be enabled");
-			equal($combo.igCombo("valueInput").attr("disabled"), undefined, "Editor should be enabled");
-			ok(!$combo.igCombo("comboWrapper").hasClass($.ui.igCombo.prototype.css.disabled), "Editor should be enabled");
-			$combo.remove();
-			$chk.remove();
-		});
+			var testId = 'Test big data';
+			test(testId, function () {
+				var $combo = $("#bigDataContainer");
+				$combo.append($("<span id='comboBigData'></span>").attr("data-bind", "igCombo: { dataSource: bigData, textKey: 'name', valueKey: 'id', width: '200', mode: 'dropdown', enableClearButton: false }"));
+				var start = (new Date()).getTime();
+				ko.applyBindings(globalViewModel, $combo[0]);
+				var end = (new Date()).getTime();
+				ok(end - start < 5000, "Binding big data is taking too much time");
```

Comment:
The problem is that ko.applyBindings, which is too slow, is not a promise, so using a timeout will not help.